x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/detail/index.tsx

```diff
@@ -396,6 +411,59 @@ export function Detail() {
     [integrationInfo, isLoading, packageInfo, fromIntegrationsPath, queryParams]
   );
 
+  const saveIntegrationEdits = async (updatedReadMe: string | undefined) => {
+    setSavingEdits(true);
+
+    const res = await updateCustomIntegration(packageInfo?.name || '', {
+      readMeData: updatedReadMe,
+      categories: [],
+    });
+
+    setSavingEdits(false);
+    setIsEditOpen(false);
+    // if everything is okay, then show success and redirect to new page
+    if (!res.error) {
+      services.notifications.toasts.addSuccess({
+        title: i18n.translate('xpack.fleet.epm.editReadMeSuccessToastTitle', {
+          defaultMessage: 'Read me updated',
+        }),
+        text: i18n.translate('xpack.fleet.epm.editReadMeSuccessToastText', {
+          defaultMessage:
+            'The read me content has been updated successfully. Redirecting you to the updated integration!',
```

Comment:
```suggestion
            'The README content has been updated successfully. Redirecting you to the updated integration.',
```
Looks good! I'd personally use a "." instead of "!", but either is really okay.